linux/docker_virtualenv_ius_apache24.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -e -u -x
+
+#OMEROVER=omero
+OMEROVER=omerodev
+
+source settings.env
+
+#install httpd 2.4
+yum -y install httpd24u
+
+#Create virtual env.
+virtualenv /tmp/omeroenv
+set +u
+source /tmp/omeroenv/bin/activate
+set -u
+
+# install omero dependencies
+/tmp/omeroenv/bin/pip2.7 install pillow numpy matplotlib
+
+# Django
+/tmp/omeroenv/bin/pip2.7 install "Django<1.9"
```

Comment:
Why just not use ``pip install -r /home/omero/OMERO.server/share/web/requirements-py27-apache.txt``. We do not need to maintain version number